builtin/common/information_formspecs.lua

```diff
@@ -51,32 +52,33 @@ core.after(0, load_mod_command_tree)
 
 local function build_chatcommands_formspec(name, sel, copy)
 	local rows = {}
-	rows[1] = "#FFF,0,Command,Parameters"
+	rows[1] = "#FFF,0,"..F(S("Command"))..","..F(S("Parameters"))
```

Comment:
(and all other `F(S())`)

```suggestion
	rows[1] = "#FFF,0,"..FS("Command")..","..FS("Parameters")
```